cluster/src/main/java/io/scalecube/cluster/membership/MembershipProtocolImpl.java

```diff
@@ -70,15 +68,14 @@
 
   // Subject
 
-  private final Subject<MembershipEvent, MembershipEvent> subject =
-      PublishSubject.<MembershipEvent>create().toSerialized();
+  private final Processor<MembershipEvent, MembershipEvent> subject = EmitterProcessor.create();
```

Comment:
FluxProcessor instead of Processor so wrapping `Flux.from` will be not needed anymore